Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -526,29 +526,33 @@ const ScrollView = createReactClass({
   },
 
   /**
-   * Scrolls to a given x, y offset, either immediately or with a smooth animation.
+   * Scrolls to a given x, y offset, either immediately, with a smooth animation, or,
+   * for Android only, a custom animation duration time.
    *
    * Example:
    *
-   * `scrollTo({x: 0, y: 0, animated: true})`
+   * `scrollTo({x: 0, y: 0, animated: true, duration: 0})`
```

Comment:
This is not a good example because it's unclear what this should do.

Reply:
Added a separate example just for duration.